platforms/unix/vm/sqUnixMain.c

```diff
@@ -1467,7 +1464,8 @@ static int vm_parseArgument(int argc, char **argv)
 
   /* vm arguments */
 
-  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage();		return 1; }
+  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage(0);         /*NOTREACHED*/}
+  else if (!strcmp(argv[0], VMOPTION("version")))	{ versionInfo();	return 0; }
```

Comment:
Why does "version" need to be added? It seems it already works...
Pharo 5 $     ./pharo --version > /dev/null ; echo $?
0
Pharo6-stable-threaded-20170503 $    ./pharo --version > /dev/null ; echo $?
0
Squeak5.1-16549-32bit-20160817 $    ./squeak.sh -version > /dev/null ; echo $?
0